backend/danswer/search/search_nlp_models.py

```diff
@@ -84,20 +84,24 @@ def build_model_server_url(
 class EmbeddingModel:
     def __init__(
         self,
-        model_name: str,
-        query_prefix: str | None,
-        passage_prefix: str | None,
-        normalize: bool,
         server_host: str,  # Changes depending on indexing or inference
         server_port: int,
+        model_name: str | None = None,
```

Comment:
Preference to have these not as defaults because it makes it easier to make mistakes in usage in other places. I think it's better to be more intentional with these. If a function has a bunch of parameters, the caller should specify them as named arguments for clarity as well